grizzly-ls/src/grizzly_ls/server/__init__.py

```diff
@@ -492,6 +494,12 @@ def initialize(ls: GrizzlyLanguageServer, params: lsp.InitializeParams) -> None:
         ls.variable_pattern = re.compile(variable_pattern)
     # // -->
 
+    # <!-- set file ignore patterns
+    file_ignore_patterns = ls.client_settings.get('file_ignore_patterns', [])
```

Comment:
`ls.file_ignore_patterns = ls.client_settings.get('file_ignore_patterns', [])`

and just skip the if-clause, default value is set to `[]` on line 116.